From akpm@osdl.org Thu Jan 19 23:59:29 2006
Message-Id: <200601200759.k0K7xPcm020721@shell0.pdx.osdl.net>
Subject: [patch 1/2] usb: usbip warning fixes
To: greg@kroah.com
Cc: akpm@osdl.org
From: akpm@osdl.org
Date: Thu, 19 Jan 2006 23:59:11 -0800


From: Andrew Morton <akpm@osdl.org>

drivers/usb/ip/stub_tx.c: In function `stub_send_txdata':
drivers/usb/ip/stub_tx.c:158: warning: int format, different type arg (arg 7)
drivers/usb/ip/vhci_tx.c: In function `vhci_send_txdata':
drivers/usb/ip/vhci_tx.c:137: warning: int format, different type arg (arg 7)


Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/usb/ip/stub_tx.c |    3 ++-
 drivers/usb/ip/vhci_tx.c |    3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/usb/ip/stub_tx.c
+++ gregkh-2.6/drivers/usb/ip/stub_tx.c
@@ -155,7 +155,8 @@ static int stub_send_txdata(struct stub_
 		int ret;
 		ret = usbip_sendmsg(sdev->ud.tcp_socket, &msg, txsize);
 		if (ret != txsize) {
-			VHCI_ERROR("vhci_sendmsg failed!, retval %d for %d\n", ret, txsize);
+			VHCI_ERROR("vhci_sendmsg failed!, retval %d for %zd\n",
+					ret, txsize);
 			usbip_event_add(&sdev->ud, SDEV_EVENT_ERROR_TCP);
 			return -1;
 		}
--- gregkh-2.6.orig/drivers/usb/ip/vhci_tx.c
+++ gregkh-2.6/drivers/usb/ip/vhci_tx.c
@@ -134,7 +134,8 @@ static int vhci_send_txdata(struct vhci_
 		int ret;
 		ret = usbip_sendmsg(vdev->ud.tcp_socket, &msg, txsize);
 		if (ret != txsize) {
-			VHCI_ERROR("vhci_sendmsg failed!, retval %d for %d\n", ret, txsize);
+			VHCI_ERROR("vhci_sendmsg failed!, retval %d for %zd\n",
+					ret, txsize);
 			usbip_event_add(&vdev->ud, VDEV_EVENT_ERROR_TCP);
 			return -1;
 		}
